exporters/opensearch-exporter/src/main/java/io/camunda/zeebe/exporter/opensearch/RestClientFactory.java

```diff
@@ -133,7 +133,7 @@ public void configureAws(
     final Aws4Signer signer = Aws4Signer.create();
 
     final HttpRequestInterceptor signInterceptor =
-        new AwsSignHttpRequestInterceptor(
+        new AwsRequestSigningApacheInterceptor(
```

Comment:
Such a seamless integration, don't even need to change the parameters 😄

Reply:
Smooth integration 😄